ci2/ci/github.py

```diff
@@ -256,10 +281,12 @@ def config(self):
             if batch and not self.batch:
                 await batch.cancel()
 
-    async def _heal(self, batch_client, run, seen_batch_ids):
+    async def _heal(self, batch_client, merge_candidate, seen_batch_ids):
```

Comment:
Sorry, I meant this should be a flag if self is the merge candidate.  Non-you merge candidate objects are none of your business.